### DISCUSSION TITLE
Garlic Crab (Hotfix 9)

### DISCUSSION THREAD

**Discussion by loathingKernel** [RareDevs/Rare#346] [Releases]
## News
Yes, we jumped a release number, it's fine. Release `1.10.8` was supposed to come a few days ago but some issues came up and commits happened.

This release should finally fix the issues Rare has had with Legendary's locking mechanism. It should also bring a number of improvements to the macOS platform which was in a really bad shape for quite some time now. There are still minor issues with it, and it lacks support for CrossOver but at least now it should be usable for native games. We have left the wine integration options enabled on macOS, but they are not tested at all so YMMV. On the other hand, Rare now has better defaults for that platform. It will also show information and details based on the current platform instead of defaulting to Windows. If you would like to help with bringing better integration with the macOS tools to run Windows games on that OS, the position it open.

If you have installed macOS native games previously through Rare, and they do not launch, you have most likely installed the Windows version of these games. The issue has been resolved and Rare will now default to installing the native version on macOS, but you will have to uninstall and re-download the games for them to work.

### Features
* Made the browser login form more readable (hopefully) and added a privacy notice.
* Added a `Quit` button in the account menu which is useful in case Rare doesn't run in a fully-fledged window manager, such as a `gamescope` session.
* Made library icons slightly smaller. This is temporary and a test if we should make them adjustable.
* Various text input fields will also show their documentation on hover as a tool-tip.
* Add a new setting to change the default installation folder for native games on macOS.
* Update the installation dialog to be easier to use.
  - Show the correct path when updating/repairing/modifying an already installed game.
  - Disable immutable options in the interface when updating/repairing/modifying an already installed game.
  - Offer the correct installation folder on macOS if the game is native.
* Validate some important to Rare settings in Legendary's configuration file at startup.
* Do not check for Legendary updates when starting the application for a slightly less startup delay.
* Default to `macOS games` filter on macOS when starting Rare for the first time.
* Add platform selection when importing games. Available only when importing a single game.
* For multiple game import the current OS will be assumed as platform and fallback to Windows if the game doesn't support it.

### Bug fixes
* Fix more issues with legendary's newly introduced file locking mechanism and how Rare uses legendary
* Fix `flatpak-spawn` to correctly export the environment. Flatpak support is still in a bad shape generally
* Fixed a bug on macOS that caused it to load only the Windows assets. This caused Rare to only offer the option to install the Windows version on macOS.
* Initialize the library once the window is visible, instead of application initialization.
* Potentially fix issues when installing through `pip` by restructuring Rare's entry points
* Respect `LEGENDARY_CONFIG_PATH` if set when creating a new configuration.
* Use the same default paths as Legendary instead of hard-coding our own everywhere.
* Do not assume `Windows` platform when importing games.
* Unreal Engine is now excluded from all other filters except for the dedicated `Include Unreal Engine` filter in the library.
* Add some more logging all around Rare.
* Hide proton settings on macOS

**Full Changelog**: https://github.com/RareDevs/Rare/compare/1.10.7...1.10.9

<hr /><em>This discussion was created from the release <a href='https://github.com/RareDevs/Rare/releases/tag/1.10.9'>Garlic Crab (Hotfix 9)</a>.</em>

**Comment by dimitris232** [comment]
how i download it

**Reply by EuSoJao** [reply]
at releases

**Comment by crtpo** [comment]
so why can't i see any game's
 that i can install?

**Reply by loathingKernel** [reply]
![image](https://github.com/RareDevs/Rare/assets/142770/83ae9585-1d17-4135-a943-b8607809ea51)

Check that the filter (the red circle) is in `All games` and that you have games associated with your account (the green circle)

If there are no games associated with your account, you can add games to it through the Epic Games Store website.
